### ISSUE TITLE
Question about the Extension of Neural Synthesis Algorithm Using TexTile

### ISSUE THREAD

**Issue by leem18** [crp94/textile#4]
Hi! Thanks for sharing such an impressive research. 

While studying your research, I tried to plug in TexTile into other models as you mentioned, "First, we extend an optimization-based neural texture synthesis algorithm to generate tileable textures. To do so, we simply optimize a joint loss function..." However, I am not sure if I did it in a right way. I simply revised function "closure()" given in the following link: https://github.com/tchambon/A-Sliced-Wasserstein-Loss-for-Neural-Texture-Synthesis

I attached the code lines I modified below. Can you please check if this aligns with what you meant in your research journal?

```
def closure():
    optimizer.zero_grad()
    loss_style  = slicing_loss(image_optimized, image_example)

    textile_loss = loss_textile(image_optimized)

    total_loss = loss_style + textile_loss
    total_loss.backward()
    return total_loss
```

Thank you!

**Comment by crp94** [comment]
Hi! Thanks for the interest in our work. Yes, that should work I think. However, the closure() step seems needed for the slicing loss. TexTile should also work without the closure(), which I believe is needed for LBFGS. TexTile works well with optimizers like Adam, which does not need the closure step. 

 I would suggest you also weight each component of the loss in a different way. 

I hope it works!
